src/scripting/KM_Scripting.pas

```diff
@@ -492,12 +493,12 @@ function TKMScripting.ScriptOnUses(Sender: TPSPascalCompiler; const Name: AnsiSt
     RegisterMethodCheck(c, 'function AbsS(aValue: Single): Single');
 
     RegisterMethodCheck(c, 'function ArrayElementCount(aElement: AnsiString; aArray: array of AnsiString): Integer');
-    RegisterMethodCheck(c, 'function ArrayElementCountB(aElement: Boolean; aArray: array of Boolean): Integer');
+    RegisterMethodCheck(c, 'function ArrayElementCountB(aElement: Boolean; aArray: TKMBooleanArray): Integer');

```

Comment:
It's not very clear if mapmakers should be using "TKMSmthArray" or "array of Smth". Let's choose one pattern and adhere to it?

Reply:
For this " temp" fix I simply changed to this.
I think changing all "array of {Type}" to TKM{Type}Array would be OK too.

I personally prefer changing to TKM{Type}Array

Reply:
I guess "array of Smth" looks simpler and more familiar to Pascal scripters

Reply:
That's true.
It's also documentation-friendly. (If people search for PS they can find array of, not TKM)
What we could do is manually implement the fix for PS.

This would solve the issue as well.

Reply:
True